### PR TITLE
(Don't use) (Broken) (Being Fixed)

### PR DESCRIPTION
**Pull request by eatdostacos**
This is untested. I wanted to change the button to say saving once you are done editing a claim. Currently once you are done editing, you click a button that says "Edit" which doesn't seem right. You have just edited. Now it should say save to save what you just edited.

## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [x] Other - Please describe: Improved sounding name.

## Fixes
Some text that I didn't feel seemed correct.
## What is the current behavior?
The button text has a weird name.
## What is the new behavior?
I hope the button now has a different name.
## Other information
I tested my repo on Linux and it works and shows the correct names.

### CODE REVIEW

File: ui/component/publishForm/view.jsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ function PublishForm(props: Props) {
 
   let submitLabel;
   if (isStillEditing) {
-    submitLabel = !publishing ? __('Edit') : __('Editing...');
+    submitLabel = !publishing ? __('Saving') : __('Saving...');
```
Comment (btzr-io) [inline ui/component/publishForm/view.jsx:106]: I think it should be `Save`  or `Update` instead of `_('Saving')`

**Comment by eatdostacos** [comment]
Ur right. Oops.

**Comment by seanyesmunt** [comment]
Thanks for the PR @eatdostacos! I agree "save" is more clear than "edit". This can be merged once you change "Saving" to "Save"

**Comment by eatdostacos** [comment]
Sorry about my grammar on the pull request description. I just got up and am not fully awake.

**Comment by tzarebczan** [comment]
Congrats on the first contribution @eatdostacos ! I know it was tough to get started, but you did it! 

Can we show you [some appreciation](htts://lbry.com/faq/appreciation)?

**Comment by eatdostacos** [comment]
Sure? I don’t know what you’re going to do but ok.

**Comment by eatdostacos** [comment]
Update: I got Linux running in a VM and the buttons now show up correctly in the build.

**Comment by eatdostacos** [comment]
Closed. Stuff got messed up and I'm doing it as a new pull request.

**Pull request closed without merging**